test/CMakeLists.txt

```diff
@@ -58,10 +58,13 @@ endif()
 enable_testing()
 
 # Note: doctest and similar testing frameworks can automatically configure CMake tests For other
-# testing frameworks add the tests target instead: ADD_TEST(GreeterTests GreeterTests)
-
-include(${doctest_SOURCE_DIR}/scripts/cmake/doctest.cmake)
-doctest_discover_tests(GreeterTests)
+# testing frameworks add the tests target instead:
+if(DEFINED doctest_SOURCE_DIR)
+  include(${doctest_SOURCE_DIR}/scripts/cmake/doctest.cmake)
+  doctest_discover_tests(GreeterTests)
+else()
+  add_test(greeterTests GreeterTests)
+endif()
```

Comment:
For the sake of simplicity I think it's ok to assume that users to either don't have `CPM_USE_LOCAL_PACKAGES` enabled, or that they know what they're doing and can adapt the starter to their specific use-case anyways. If we would try to support every possible use-case in the starter it would become increasingly complex to use.

Reply:
that is a real problem, it does not work if the dockets was found on build host: the include fails!